enioka_scan_zebra_dw/README.md

```diff
@@ -1,32 +0,0 @@
-# Zebra DataWedge provider
```

Comment:
This file contains interesting information, it may be worth keeping it maybe inside the documentation's `dependencies` page in the zebra-dw section to make sure it is not lost.

Reply:
I merged some of these informations into the readthedoc documentations, but I will undo the deletion.